sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -1339,13 +1353,29 @@ public void importBatch(FinishBundleContext context) throws IOException {
       this.fhirStore = fhirStore;
     }
 
-    /**
-     * Instantiates a new Execute bundles.
-     *
-     * @param fhirStore the fhir store
-     */
-    ExecuteBundles(String fhirStore) {
-      this.fhirStore = StaticValueProvider.of(fhirStore);
+    @Override
+    ValueProvider<String> getFhirStore() {
+      return null;
+    }
+
+    @Override
+    WriteMethod getWriteMethod() {
+      return WriteMethod.EXECUTE_BUNDLE;
+    }
+
+    @Override
+    Optional<ContentStructure> getContentStructure() {
+      return Optional.empty();
+    }
+
+    @Override
+    Optional<ValueProvider<String>> getImportGcsTempPath() {
+      return Optional.empty();
+    }
+
+    @Override
+    Optional<ValueProvider<String>> getImportGcsDeadLetterPath() {
+      return Optional.empty();
```

Comment:
do you use ValueProviders / templates in your use case? If you do not, then I think it does not make sense to add new properties that depend on value providers. Instead, just make this Optional<String>

Reply:
I'm not sure what you mean, these methods are only here because I made ExecuteBundles implement FhirIO.Write, which requires them.